packages/hunspell.rb

```diff
@@ -14,9 +14,9 @@ class Hunspell < Package
   if ARGV[0] == 'install'
```

Comment:
Use `def self.preflight` instead?

Reply:
I don't think so, because self.preflight gets run every single time during `crew update`

Reply:
Good point.  See #5528.

Reply:
Can we change line 14 to `def self.preflight` since #5528?